pinot-core/src/test/resources/data/text_search_data/skills.txt

```diff
@@ -21,4 +21,7 @@ C++, Java, Python, realtime streaming systems, Machine learning, spark, Kubernet
 Databases, columnar query processing, Apache Arrow, distributed systems, Machine learning, cluster management, docker image building and distribution
 Database engine, OLAP systems, OLTP transaction processing at large scale, concurrency, multi-threading, GO, building large scale systems
 GET /administrator/ HTTP/1.1 200 4263 - Mozilla/5.0 (Windows NT 6.0; rv:34.0) Gecko/20100101 Firefox/34.0 - NullPointerException
-Foo worked in a lot of places and learned a lot of things
\ No newline at end of file
+Foo worked in a lot of places and learned a lot of things
+IT support, python, hardware debugging
+IT staff, workspace coordinator
```

Comment:
Can we verify that without this fix, query that uses 'IT support OR IT Manager' will return the count same as 'support OR Manager's

I think the test data needs further change.

Reply:
Done